### DISCUSSION TITLE
Any suggestions for settings without lazy.nvim?

### DISCUSSION THREAD

**Discussion by GaryLiuTelus** [folke/flash.nvim#205] [Q&A]
I am beginner of vim, and have a configuration using Packer to manage my extensions.  On the main page, there is only an example for settings using lazy.nvim. Anyone can suggest a example settings I can follow using Packer? 

I know lazy.nvim has lots advantages over Packer, but it might means too many changes to me to migrate from packer to lazy.nvim. 

Thanks!

**Comment by IndianBoy42** [comment]
lazy.nvim has help on transitioning from packer, the same could be reversed. In general just understand what the example config is doing, find how to do it in your plugin manager and then translate it.

**Comment by jgregoire** [comment]
```
use { 'folke/flash.nvim' }

-- Somewhere else:
require('flash').setup({
    -- Put your settings here
})
```
